forge-gui/res/cardsfolder/upcoming/trazyn_the_infinite.txt

```diff
@@ -0,0 +1,6 @@
+Name:Trazyn the Infinite
+ManaCost:4 B B
+Types:Legendary Artifact Creature Necron
+PT:4/6
+S:Mode$ Continuous | Affected$ Card.Self | EffectZone$ Battlefield | GainsAbilitiesOf$ Artifact.YouOwn | GainsAbilitiesOfZones$ Graveyard | Description$ As long as Trazyn the Infinite is on the battlefield, it has all activated abilities of all artifact cards in your graveyard.
+Oracle:As long as Trazyn the Infinite is on the battlefield, it has all activated abilities of all artifact cards in your graveyard.

```

Comment:
`DeckHints:Type$Artifact & Ability$Graveyard` 
Could make an argument to upgrade that to DeckNeeds